ui/apps/platform/src/Containers/SystemHealth/Components/IntegrationHealthWidgetVisual.tsx

```diff
@@ -1,40 +1,75 @@
 import React, { ReactElement } from 'react';
-import { Message } from '@stackrox/ui-components';
+import {
+    Card,
+    CardBody,
+    CardHeader,
+    CardHeaderMain,
+    CardTitle,
+    Flex,
+    FlexItem,
+} from '@patternfly/react-core';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
 
-import { integrationsPath } from 'routePaths';
-import ViewAllButton from 'Components/ViewAllButton';
-import Widget from 'Components/Widget';
+import pluralize from 'pluralize';
+import { Message } from '@stackrox/ui-components';
 import IntegrationsHealth from './IntegrationsHealth';
 import { IntegrationMergedItem } from '../utils/integrations';
+import IconText from '../../../Components/PatternFly/IconText/IconText';
 
 type IntegrationHealthWidgetProps = {
-    id: string;
     integrationText: string;
     integrationsMerged: IntegrationMergedItem[];
     requestHasError: boolean;
 };
 
 const IntegrationHealthWidget = ({
-    id,
     integrationText,
     integrationsMerged,
     requestHasError,
 }: IntegrationHealthWidgetProps): ReactElement => {
+    const integrations = integrationsMerged.filter((integrationMergedItem) => {
+        return integrationMergedItem.status === 'UNHEALTHY';
+    });
     return (
-        <Widget
-            header={integrationText}
-            headerComponents={<ViewAllButton url={`${integrationsPath}#${id}`} />}
-            className="h-48 text-lg"
-            id={id}
-        >
-            {requestHasError ? (
-                <div className="p-2 w-full">
+        <Card isFullHeight isCompact>
+            <CardHeader>
+                <CardHeaderMain>
+                    <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                        <FlexItem>
+                            <CardTitle component="h2">{integrationText}</CardTitle>
+                        </FlexItem>
+                        {!requestHasError && (
+                            <FlexItem>
+                                <IconText
+                                    icon={
+                                        integrations.length === 0 ? (
+                                            <CheckCircleIcon color="var(--pf-global--success-color--100)" />
+                                        ) : (
+                                            <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+                                        )
+                                    }
+                                    text={
+                                        integrations.length === 0
+                                            ? 'no errors'
+                                            : `${integrations.length} ${pluralize(
+                                                  'error',
+                                                  integrations.length
+                                              )}`
+                                    }
+                                />
```

Comment:
Here is pattern to replace `IconText` element at right of title:
* with `icon` at left: https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/SystemHealth/DeclarativeConfigurationHealth/DeclarativeConfigurationHealthCard.tsx#L58-L64
* and count-noun phrase at right: https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/SystemHealth/DeclarativeConfigurationHealth/DeclarativeConfigurationHealthCard.tsx#L77-L79